tools/deflatepack.c

```diff
@@ -34,14 +34,14 @@ static struct SizedBuffer readFile(const char* filename, int margin)
     }
 
     fseek(in, 0, SEEK_END);
-    int size = ftell(in);
+    size_t size = ftell(in);
```

Comment:
nit: `size` can be negative on fail, use a signed `long` (and update the print below accordingly)